order-book/src/test/java/com/market/banica/order/book/service/grpc/componentTests/OrderBookComponentIT.java

```diff
@@ -0,0 +1,227 @@
+package com.market.banica.order.book.service.grpc.componentTests;
+
+import com.aurora.Aurora;
+import com.aurora.AuroraServiceGrpc;
+import com.market.Origin;
+import com.market.TickResponse;
+import com.market.banica.order.book.OrderBookApplication;
+import com.market.banica.order.book.exception.TrackingException;
+import com.market.banica.order.book.service.grpc.AuroraClient;
+import com.market.banica.order.book.service.grpc.OrderBookService;
+import com.market.banica.order.book.service.grpc.componentTests.configuration.TestConfiguration;
+import com.orderbook.CancelSubscriptionRequest;
+import com.orderbook.CancelSubscriptionResponse;
+import com.orderbook.InterestsRequest;
+import com.orderbook.InterestsResponse;
+import com.orderbook.ItemOrderBookRequest;
+import com.orderbook.ItemOrderBookResponse;
+import com.orderbook.OrderBookLayer;
+import com.orderbook.OrderBookServiceGrpc;
+import io.grpc.Context;
+import io.grpc.ManagedChannel;
+import io.grpc.StatusRuntimeException;
+import io.grpc.inprocess.InProcessChannelBuilder;
+import io.grpc.inprocess.InProcessServerBuilder;
+import io.grpc.stub.StreamObserver;
+import io.grpc.testing.GrpcCleanupRule;
+import org.junit.Rule;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.ArgumentCaptor;
+import org.mockito.ArgumentMatchers;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.SpyBean;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;
+
+import java.io.IOException;
+import java.util.concurrent.Executors;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.doReturn;
+import static org.mockito.Mockito.verify;
+
+@SpringJUnitConfig
+@SpringBootTest(classes = OrderBookApplication.class, webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@RunWith(JUnit4.class)
```

Comment:
could you please check if other services are using JUnit5 (I think Radoslav was working on this) and discuss with the team which version you are going to use and have all tests implemented consistently

Reply:
Fixed